docs/docs/docs/01-core/admin/01-ee/00-intro.md

```diff
@@ -35,7 +35,7 @@ Everytime a new EE feature is added in Strapi, in the settings menu, you should
             to: '/settings/purchase-new-ee-feature',
             id: 'new-ee-feature',
             // TODO: to replace with another name in v5
-            lockIcon: true,
+            toBuy: true,
```

Comment:
wouldn't `enterpriseOnly` be more appropriate?

Reply:
it could be a good name, tbh it can be served also with other cloud licenses, maybe we can use something like `paidLicenseOnly` or just `licenseOnly`, wdyt?

Reply:
>tbh it can be served also with other cloud licenses

That's true but we don't really make that distinction inside the codebase, I don't think we need to start worrying about it now. For example `admin/ee` hosts most of the code that's protected by a license, no matter if it's in the cloud plans or not

Reply:
`licenseOnly` makes sense.

Reply:
ok thanks